### PR TITLE
feature virtual training series

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Viewed the changes in RTD both in the left hand menu and on this page.
https://metplus-training.readthedocs.io/en/feature_virtual_training_series/modules/Tutorial/index.html

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the changes.

- [x] Please complete this pull request review by **[ASAP]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Linked issues** with the original issue number. (**N/A**)
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/modules/Tutorial/index.rst [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+Virtual and Online Tutorial Topics
+==================================
 
 The following topics are provide in support of the `METplus Online Tutorial <http://dtcenter.org/community-code/metplus/online-tutorial>`_.
 
+***********************
+Virtual Training Series
+***********************
+
+Recording and presentation provided during the 2021-2022 METplus Training
```
Comment (JohnHalleyGotway) [inline docs/modules/Tutorial/index.rst]:
```suggestion
Recordings and presentations provided during the 2021-2022 METplus Training
```

File: docs/modules/Tutorial/index.rst [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+Series may be found on the
+`Training Series Agenda <https://dtcenter.org/events/2021/metplus-training-series/agenda>`_.
+The training series leverages the METplus Online Tutorial for the hands-on
```
Comment (JohnHalleyGotway) [inline docs/modules/Tutorial/index.rst]:
```suggestion
The training series leverages the [METplus Online Tutorial](https://dtcenter.org/metplus-practical-session-guide-version-4-0/session-1-metplus-setupgrid-grid) for the hands-on
```

**Review by JohnHalleyGotway — APPROVED** [review]
I approved of this PR. The changes look good to me and match what we discussed previously.

Do note that I proposed 2 suggested minor edits.

**Comment by jprestop** [comment]
Thanks, @JohnHalleyGotway.  I made the changes you suggested.  I'm not sure why the checks have failed, so I'll look into that.

**Comment by jprestop** [comment]
@JohnHalleyGotway  The warning in the docs is:

> /home/runner/work/METplus-Training/METplus-Training/docs/modules/Tutorial/index.rst:3: WARNING: Duplicate explicit target name: "metplus online tutorial".

The METplus Online Tutorial is already linked on that same page at the top of the page and Sphinx doesn't like links with the same name, so I think I'll remove the link I just added (your second suggestion) if that is ok with you?  We could also potentially rename it, but I couldn't think of anything good to rename it.

**Pull request merged**